ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -0,0 +1,387 @@
+              onRoleChange={setRole}
+              disabled={isEnrollmentLoading}
+            />
+          </div>
+          <Typography className={classes.spacing}>
+            Select the section you want to enroll the user in.
+          </Typography>
+          {
+            showIncompleteAlerts && selectedSection === undefined &&
+              <InlineErrorAlert>You must select one section from the list below.</InlineErrorAlert>
+          }
+          <SectionSelectorWidget
+            height={300}
+            search={[]}
+            multiSelect={false}
+            sections={props.sections}
+            selectedSections={selectedSection !== undefined ? [selectedSection] : []}
+            selectionUpdated={(sections) => {
+              if (sections.length === 0) {
+                setSelectedSection(undefined)
+              } else {
+                setSelectedSection(sections[0])
+              }
+            }}
+            canUnmerge={false}
+          />
+          <Grid container className={classes.buttonGroup} justifyContent='space-between'>
+            <Button
+              variant='outlined'
+              aria-label='Back to input method select'
+              onClick={props.resetFeature}
+            >
+              Back
+            </Button>
+            <Button
+              color='primary'
+              variant='contained'
+              disabled={isLoading}
+              aria-label='Submit single user enrollment'
+              onClick={handleSubmitClick}
+            >
+              Submit
+            </Button>
+          </Grid>
+        </Grid>
+        <Backdrop className={classes.backdrop} open={isEnrollmentLoading}>
+          <Grid container>
+            <Grid item xs={12}>
+              <CircularProgress color='inherit' />
+            </Grid>
+            <Grid item xs={12}>Sending user and enrollment data to Canvas</Grid>
+          </Grid>
+        </Backdrop>
+      </Grid>
+    )
+  }
+
+  const renderSuccess = (userExists: boolean): JSX.Element => {
+    const settingsLink = (
+      <Link href={props.settingsURL} target='_parent'>Canvas Settings page</Link>
+    )
+    const messageText = (
+      userExists
+        ? 'The existing user was'
+        : 'The new user was invited to create a friend account, added to Canvas, and'
+    ) + ' enrolled in the selected section!'
+    const nextAction = (
+      <span>See the user in the course&apos;s sections on the {settingsLink} for your course.</span>
+    )
+
+    return (
+      <>
+      <SuccessCard message={<Typography>{messageText}</Typography>} nextAction={nextAction} />
+      <Grid container className={classes.buttonGroup} justifyContent='flex-start'>
+        <Button variant='outlined' onClick={props.resetFeature}>Start Again</Button>
+      </Grid>
+      </>
+    )
+  }
+
+  return (
+    <div id='single-add-user'>
+      <Typography variant='h6' component='h2' gutterBottom>Add Single User Manually</Typography>
```

Comment:
I feel there must be tinny line gap between feature title and feature option for adding single or multiple user
![Screen Shot 2022-01-06 at 4 07 28 PM](https://user-images.githubusercontent.com/8579775/148452405-57b2b372-6992-49dd-8687-73dadea2f436.png)

Reply:
Do you want the spacing/gap to be larger?

Reply:
just a smidgen

Reply:
See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/commits/7b3a2aa7fc33923d08fff9acfd759551c44acc83

We should also consider adding similar spacing below the other `h1`s, but not sure if that should be done here.

I found I wasn't using this CSS class anymore, so I'm going to ensure I didn't leave any other stranded classes in other components.

Reply:
>We should also consider adding similar spacing below the other h1s, but not sure if that should be done here.

I think the gap is not obvious with other features since none as of now as sub-selection. So Add UM will have that if you resolve #278